web/themes/custom/fmcg_material/sass/styles.scss

```diff
@@ -1727,7 +1554,7 @@ label.option {
 }
 
 nav.tabs {
-  max-width: 1200px;
+  max-width: 360px;
```

Comment:
I'm guessing this is for the benefit of the login/register page? We need to make sure that only applies to these pages.